savoirfairelinux/sous-chef
# #381 add django-avatar so  users can select a profile picture

### Changes proposed in this pull request:

*  use package django-avatar in order to handle avatar upload files
*  if nothing uploaded will try to use gravatar according user email 
* I override the default upload page to extend sous-chef base page 
* all avatar pics are uploaded in the media folder

### Status


### How to verify this change
Just run the application, profile picture should be the user gravatar if nothing a default picture is displayed.
Click on the avatar picture to be redirected to the management page, you can upload a new pics and also delete old ones.  

### Deployment notes and migration

### Additional notes

A new dependency has been added. The containers must be rebuild.
